examples/azure-rules.yaml

```diff
@@ -656,4 +656,17 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
-        scored: true
\ No newline at end of file
+        scored: true
+      - id: 1.5.7
+        description: "Check /hana/share/myHooks/global.ini"
+        audit: 'cat /hana/share/myHooks/global.ini'
+        tests:
+          test_items:
+          - flag: "provider = SAPHanaSR"
+          - flag: "path = /usr/share/SAPHanaSR"
+          - flag: "execution_order = 1"
+          - flag: "ha_dr_saphanasr = info"
+        remediation: |
+          Set the recommended values in global.ini. Ref: section 8.1 Implementing the Python Hook SAPHanaSR
+          https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
```

Comment:
is this really an "azure check"? shouldn't we refer to https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#implement-the-python-system-replication-hook-saphanasr then instead?